### ISSUE TITLE
All-uppercase names especially highlighted

### ISSUE THREAD

**Issue by Siguza** [phalcon/zephir-atom#3]
All-uppercase names receive a special highlighting (bold orange-yellow for me).

Try with:
```
class ALLUPPERCASENAME
{}
```

Below is the generated HTML from Atom:
```
<span class="source js">
    <span class="storage type js">class</span>
    <span class="string regexp js">ALLUPPERCASENAME</span>
    <span class="meta brace curly js">{}</span>
</span>
```

**Comment by nkt** [comment]
Because in Zephir all uppercase named variables are constants.

**Comment by Siguza** [comment]
Alright, gives a syntax error when used as class name. Still:

`namespace some\UPPERCASE\namespace;`

**Comment by nkt** [comment]
Yep. That happens because Zephir parser recognize `[A-Z][A-Z0-9_]*` as constants.

**Comment by Siguza** [comment]
Isn't it `[A-Z_][A-Z0-9_]*`?
Anyway, the actual Zephir parses allows those pattern in namespaces, so I think the atom syntax should reflect that, right?
